src/components/Logo.tsx

```diff
@@ -1,13 +1,20 @@
-export default function Example() {
+import { useState } from "react"
+
+export const Logo = () => {
+    const [logo, setLogo] = useState(true);
+
+    const clicked = () => {
+        setLogo(!logo);
+    }
+
     return (
         <>
             <div>
                 <div className="flex items-center justify-center relative">
                     <div className="p-1 z-50">
-                        {/* <img className="w-24" src="./EMURRCircuit.svg" /> */}
+                        <img className="w-24" src={`${logo ? "/images/EMURRPie.svg" : "/images/EMURRCircuit.svg"}`} onClick={clicked} />
                     </div>
-                    <div className="">hello</div>
```

Comment:
i would add `cursor-pointer` to:
`<img className="w-24" src={`${logo ? "/images/EMURRPie.svg" : "/images/EMURRCircuit.svg"}`} onClick={clicked} />`

so that when people hover it they know it is a clickable item... unless if we want it to be super suuuuuper secret hmm... surprise me

Reply:
i can't seem to figure out why it slightly moves i'm looking into it rn

Reply:
so i think it has to do with the SVG itself. there's no weird margins or uneven padding. just double check the svg generation to make sure they're both center relative to one another.

Reply:
Couldn't find a way to fix the images. And want to work on higher priority stuff right now. I would say add to bug fixes and then merge for now?

Reply:
for sure. i approved so it should be good to merge. if you want i can even put it on my standup to do it tomorrow. i'm v comfortable with the styling it was just the API stuff i got cold feet about.

Reply:
merging now